addon/adapters/pouch.js

```diff
@@ -340,7 +340,16 @@ export default DS.RESTAdapter.extend({
   },
 
   queryRecord: function(store, type, query) {
-    return this.query(store, type, query);
+    return this.query(store, type, query).then(results => {
+      let result = {};
```

Comment:
You need to edit the original `results` object. Since this can have additional entries, which are sideloaded.